lib/sneakers/worker.rb

```diff
@@ -120,6 +124,25 @@ def worker_trace(msg)
       logger.debug(log_msg(msg))
     end
 
+    def before_work(delivery_info, metadata, msg)
+      wbefore = Sneakers::CONFIG[:hooks][:before_work]
```

Comment:
Hey @fbbergamo why not rename this `wbefore` and the `wafter` to `hook`, wbefore and wafter sounds weird in the code